packages/jest-test-typescript-parser/package.json

```diff
@@ -9,6 +9,9 @@
   "main": "build/index.js",
   "dependencies": {
     "jest-editor-support": "^21.2.0",
-    "typescript": "^2.2.1"
+    "typescript": "^2.5.3"
+  },
+  "devDependencies": {
+    "typescript": "^2.5.3"
   }
```

Comment:
TypeScript is now in both deps + dev deps, I think it's fine in just deps 👍

Reply:
that was dumb, fixed